docs/INSTALL.md

```diff
@@ -65,7 +65,7 @@
 
     On Mac OS X:
     ```
-    brew install go
+    brew install go@1.11
```

Comment:
brew will install `v1.11` or newer already?

Reply:
I don't think it will if the user hasn't upgraded the "kegs" (or w/e they're called) for some time.

Reply:
`brew update && brew install go`? 😛